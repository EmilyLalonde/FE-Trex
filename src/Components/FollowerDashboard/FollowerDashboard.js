import React, { Component } from "react";
import Home from '../Home/Home'
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity,
  TextInput,
  Keyboard
} from "react-native";

export default class FollowerDashboard extends Component {
  constructor(props) {
    super(props)
    this.state = {

    }
  }
  render() {
    const {navigate} = this.props.navigation;
    return (
      <View>
        <Text>Display Wanderers that user follows</Text>
      </View>
    )
  }
}