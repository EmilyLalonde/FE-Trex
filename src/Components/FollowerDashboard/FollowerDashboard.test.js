import React, {Component} from 'react';
import { FollowerDashboard } from './FollowerDashboard';

describe('FollowerDashboard', () => {
  it('should return true', () => {
    expect(true).toEqual(true)
  })
})