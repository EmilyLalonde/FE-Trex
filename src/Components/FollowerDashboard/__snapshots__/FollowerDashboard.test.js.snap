// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FollowerDashboard should match the snapshot when loading 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <Header />
  <ScrollViewMock>
    <Text
      style={
        Object {
          "color": "black",
          "fontSize": 30,
          "marginVertical": 40,
          "textAlign": "center",
          "width": "auto",
        }
      }
    >
      My Wanderers
    </Text>
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#84183B",
          "borderColor": "white",
          "borderRadius": 8,
          "borderStyle": "solid",
          "borderWidth": 1,
          "color": "white",
          "fontSize": 30,
          "height": 60,
          "margin": 20,
          "padding": 10,
          "textAlign": "center",
          "width": "auto",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 20,
            "paddingVertical": 10,
            "textAlign": "center",
          }
        }
      >
        Fake Wanderer 1
      </Text>
    </TouchableOpacity>
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
      style={
        Object {
          "alignItems": "stretch",
          "backgroundColor": "#84183B",
          "borderColor": "white",
          "borderRadius": 8,
          "borderStyle": "solid",
          "borderWidth": 1,
          "color": "white",
          "fontSize": 30,
          "height": 60,
          "margin": 20,
          "padding": 10,
          "textAlign": "center",
          "width": "auto",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 20,
            "paddingVertical": 10,
            "textAlign": "center",
          }
        }
      >
        Fake Wanderer 2
      </Text>
    </TouchableOpacity>
  </ScrollViewMock>
</View>
`;
